services/master/controllers/deploy.go

```diff
@@ -118,8 +119,8 @@ func Deploy(c *fiber.Ctx) error {
 			res := make([][]string, 0)
 			for i := 0; i < int(numberOfTeams); i++ {
 				fmt.Println("-----------Deploying challenge for team: " + strconv.Itoa(i) + " --------")
-				team_name := "team-" + strconv.Itoa(i)
-				deployer.DeployChallenge(foldername, team_name, 1)
+				team_name := "katana-team-" + strconv.Itoa(i)
+				utils.DeployChallenge(foldername, team_name, false, 1)
```

Comment:
Naming convention and don't pass values as arguments, make them into variables and pass them